README.md

```diff
@@ -1,11 +1,14 @@
 # Pineapple Lock Screen
 
+Turn the screen off without using the physical power button.
+
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+     alt="Get it on F-Droid"
+     height="80">](https://f-droid.org/packages/net.blumia.pineapple.lockscreen.oss/)
 [<img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png"
       alt="Get it on Google Play"
       height="80">](https://play.google.com/store/apps/details?id=net.blumia.pineapple.lockscreen)
 
-Turn the screen off without using the physical power button.
-
 ## Pull requests
 
 I will not take any pull requests.
```

Comment:
I guess you already seen this part while editing the README.md file. For this project, I don't take Pull Request. Sorry about this.